frontend/src/state/reducers/userPlansReducer.ts

```diff
@@ -157,8 +158,16 @@ export const userPlansReducer = (
       }
       case getType(setActivePlanMajorAction): {
         const { major } = action.payload;
+        const activePlan = draft.plans[draft.activePlan!];
+
+        activePlan.major = major;
+        activePlan.concentration = null;
```

Comment:
Should coopCycle also be set to null?

Reply:
Ik we didn't have this before but we probably should lol, we do it in the setUserMajorAction